src/ros_commo.py

```diff
@@ -444,10 +445,8 @@ def __init__(self):
 			self.language_perceived_text_cb)
 
 		# Chat infrastructure text.
-		# from chatbot.msg import ChatMessage
-		#rospy.Subscriber("/robot/speech", chatbot/ChatMessage,
-		#rospy.Subscriber("chatbot_speech", chatbot/ChatMessage,
-		#	self.chat_perceived_text_cb)
+		rospy.Subscriber("chatbot_speech", ChatMessage,
```

Comment:
It is available publicly just like blender_api_msgs. You can find it in HEAD/src/chatbot/msg. Whenever you can use blender_api_msgs, you can use ChatMessage. So for user, the requirement is met already. 

Yeah,  the confidence in the ChatMessage is of no use because we simply don't set it. 

We can do it better but it's not easily to change it at this moment. Is it a good practice to have a message package that includes all the message used?